tests/homepage.spec.ts

```diff
@@ -1,12 +1,5 @@
 import { expect, test } from "@playwright/test";
 
-test("front page has welcome text", async ({ page }) => {
```

Comment:
I think this was the original playwright test just to get started. It's not too useful now :)